### ISSUE TITLE
Typography control returns both 'subset' and 'subsets' indexes with the same value

### ISSUE THREAD

**Issue by vladcosorg** [themeum/kirki#948]
### Issue description:

get_theme_mod returns both 'subset' and 'subsets' indexes
![selection_003](https://cloud.githubusercontent.com/assets/467230/14915483/1db72986-0e1c-11e6-8496-7906a66ac5e1.png)
but should have the same structure as default value

Because of that, this solution https://github.com/aristath/kirki/issues/537#issuecomment-201911111 does not work
### Version used:

(Did you try using the develop branch from github? There's a chance your issue has already been adressed there)
Version 2.3.1
### Using theme_mods or options?

theme_mods
### Code to reproduce the issue (config + field(s))

```
[
                'settings' => 'layout_color_example1',
                'label' => esc_html__('Example 1', 'cf47rs'),
                'type' => 'typography',
                'default' => [
                    'font-family' => 'Roboto',
                    'variant' => 'regular',
                    'font-size' => '14px',
                    'line-height' => '1.5',
                    'letter-spacing' => '0',
                    'subset' => ['latin-ext'],
                    'color' => '#333333',
                    'text-transform' => 'none',
                    'text-align' => 'left'
                ],
                'output' => [
                    [
                        'element' => '.variable-top-border',
                        'property' => 'border-top-width',
                        'units' => 'px',
                    ],
                    [
                        'element' => '.variable-bottom-border',
                        'property' => 'border-bottom-width',
                        'units' => 'px',
                    ],
                ]

            ]
```

**Comment by aristath** [comment]
great catch!

Fixed, in 2.3.2 `subset` becomes `subsets`.
Pending docs update on https://github.com/aristath/kirki/pull/950 as well, docs will be updated once 2.3.2 is released.

**Comment by vladcosorg** [comment]
Thank you, this plugin is awesome!
